tests/atomistics/structure/test_analyse.py

```diff
@@ -185,6 +185,16 @@ def test_tessellations(self):
             structure.positions[structure.analyse.get_voronoi_neighbors()], axis=-2
         )), axis=-1).flatten().max(), a_0)
 
+    def test_cluster_positions(self):
+        bulk = StructureFactory().ase.bulk('Fe', cubic=True)
+        self.assertEqual(len(bulk.analyse.cluster_positions()), len(bulk))
+        positions = np.append(bulk.positions, bulk.positions, axis=0)
+        self.assertEqual(len(bulk.analyse.cluster_positions(positions)), len(bulk))
+        self.assertEqual(
+            bulk.analyse.cluster_positions(np.zeros((2, 3)), return_labels=True)[1].tolist(),
+            [0, 0]
+        )
+
 
 if __name__ == "__main__":
-    unittest.main()
+    unittest.main()
```

Comment:
We actually no longer (never?) need this whole clause at the end. The CI works fine without it, and I've never run into trouble running the tests locally with out it either. I'd just cut it.

Reply:
otherwise it doesn't work on my terminal

Reply:
I agree with @samwaseda We should have it in every test class to be compatible to executing the tests on the command line.